src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitObserver.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.shortcircuit.server.service;
+
+import com.powsybl.shortcircuit.ShortCircuitAnalysis;
+import com.powsybl.shortcircuit.ShortCircuitAnalysisResult;
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+import lombok.NonNull;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+@Service
+public class ShortCircuitObserver {
+    private static final String OBSERVATION_PREFIX = "app.computation.";
+    private static final String TYPE_TAG_NAME = "type";
+    private static final String STATUS_TAG_NAME = "status";
+    private static final String COMPUTATION_TYPE = "shortcircuit";
```

Comment:
this won't work for queue metrics
this need to be the same string as the queue name
2 solution :
- rename this to "shortcircuitanalysis" -> then change grafana config according to this change
- rename all queue related to shortcircuit to "shortcircuit.*"

Reply:
will be done when rabbitmq metrics are added